src/js/covidMap.js

```diff
@@ -420,34 +499,78 @@ fluid.defaults("fluid.covidMap.map", {
                 }
             }
         },
-        selectedHospitalPane: {
-            type: "fluid.covidMap.hospitalRenderer",
-            container: "{that}.dom.hospitalPanel",
+        filterCountOnDesktop: {
+            type: "fluid.covidMap.filterCount",
+            container: "{that}.dom.filterCountOnDesktop"
+        },
+        filterCountOnMobile: {
+            type: "fluid.covidMap.filterCount",
+            container: "{map}.dom.filterCountOnMobile"
+        },
+        locationButtonOnMobile: {
+            type: "fluid.button",
+            container: "{map}.dom.locationButtonOnMobile",
             options: {
-                gradeNames: "fluid.viewComponent",
-                model: {
-                    row: "{map}.model.selectedHospital"
-                },
-                selectors: {
-                    hospitalWebsite: ".fl-mapviz-hospital-website" // This field left over from fluid.covidMap.hospitalRenderer
+                modelListeners: {
+                    "resetQuery": {
+                        path: "activate",
+                        changePath: "{map}.model.query",
+                        value: ""
+                    }
                 },
                 modelRelay: {
-                    hospitalWebsite: {
-                        source: {
-                            segs: ["row", "{map}.options.fields.website"]
-                        },
-                        target: "dom.hospitalWebsite.text"
+                    showCitiesList: {
+                        source: "activate",
+                        target: "{map}.model.visiblePanelOnMobileFlags",
+                        func: x => {
+                            if (x > 0) {
+                                return {
+                                    "citiesList": true,
+                                    "resultsPage": false,
+                                    "filterPanel": false,
+                                    "hospitalPanel": false
+                                };
+                            };
+                        }
+                    },
+                    expandCitiesList: {
+                        source: "activate",
+                        target: "{citiesList}.expandButton.model.expanded",
+                        func: x => {
+                            if (x > 0) {return !!x;}
+                        }
                     }
                 }
             }
         },
-        filterCountOnDesktop: {
-            type: "fluid.covidMap.filterCount",
-            container: "{that}.dom.filterCountOnDesktop"
-        },
-        filterCountOnMobile: {
-            type: "fluid.covidMap.filterCount",
-            container: "{that}.dom.filterCountOnMobile"
+        filterButtonOnMobile: {
+            type: "fluid.button",
+            container: "{map}.dom.filterButtonOnMobile",
+            options: {
+                modelRelay: {
```

Comment:
This is more idiomatically a changePath listener - 
e.g. 
````
modelListeners: {
    "showFilterPanel: {
        source: "activate",
        changePath: "{map}.model.visiblePanelOnMobileFlags",
        value: {
            "citiesList": false,
            "resultsPage": false,
            "filterPanel": true,
            "hospitalPanel": false
      },
      excludeSource: "init"
````
Given the mutual exclusion between the flags it would be better to model this as a single selection field plus a relay to the flags but unfortunately we don't have such a transform currently implemented - fluid.transforms.arrayToSetMembership expects a multiple selection. I've written up this issue as https://issues.fluidproject.org/browse/FLUID-6612